tests/dashbio_demos/app_ideogram.py

```diff
@@ -802,12 +781,12 @@ def ideo_homology(
             stop_two_a
     ):
         return {
-            "chr_one": {
+            "chrOne": {
```

Comment:
Why aren't we using snake case here?

Reply:
Because the prop names in the `.react.js` file are in camel case